Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/ViewUpdaters.fs

```diff
@@ -723,4 +730,20 @@ module ViewUpdaters =
         | ValueSome prevValue, ValueSome currValue when prevValue = currValue -> ()
         | ValueNone, ValueNone -> ()
         | _, ValueSome currValue -> Element.SetMenu(target, currValue.Create() :?> Menu)
-        | ValueSome _, ValueNone -> Element.SetMenu(target, null)
\ No newline at end of file
+        | ValueSome _, ValueNone -> Element.SetMenu(target, null)
+
+    let updateIndicatorViewRef (prevValueOpt: ViewRef<CustomCarouselView> voption) (currValueOpt: ViewRef<CustomCarouselView> voption) (target: Xamarin.Forms.IndicatorView) =
```

Comment:
Name should contain the property updated. So `updateIndicatorViewItemsSourceBy`.